examples/python/using_carrays.py

```diff
@@ -24,8 +24,11 @@
 
```

Comment:
Not sure what this was ever used for.  If it's just there to show how to use a swig carray type then I agree.